--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -79,16 +79,16 @@
 
 # For creating the BINARY package we include the files we want
 
-INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/dsk/bin/macosx/dsk      DESTINATION ./thirdparty/dsk/bin/macosx/)
-INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/dsk/bin/linux/dsk       DESTINATION ./thirdparty/dsk/bin/linux/)
-INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/c1.fasta.gz                   DESTINATION ./data/)
-INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/c2.fasta.gz                   DESTINATION ./data/)
-INSTALL (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/doc                            DESTINATION .)
-INSTALL (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/test                           DESTINATION .)
-INSTALL (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/scripts                        DESTINATION . FILES_MATCHING REGEX ".*\\.(py|sh)$"  PATTERN "jenkins" EXCLUDE)
-INSTALL (FILES     ${CMAKE_CURRENT_SOURCE_DIR}/short_read_connector.sh        DESTINATION .)
-INSTALL (FILES     ${CMAKE_CURRENT_SOURCE_DIR}/README.md                      DESTINATION .)
-INSTALL (FILES     ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE                        DESTINATION .)
+# INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/dsk/bin/macosx/dsk      DESTINATION ./thirdparty/dsk/bin/macosx/)
+# INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/dsk/bin/linux/dsk       DESTINATION ./thirdparty/dsk/bin/linux/)
+# INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/c1.fasta.gz                   DESTINATION ./data/)
+# INSTALL (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/c2.fasta.gz                   DESTINATION ./data/)
+# INSTALL (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/doc                            DESTINATION .)
+# INSTALL (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/test                           DESTINATION .)
+INSTALL (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/scripts                        DESTINATION share/short-read-connector/ FILES_MATCHING REGEX ".*\\.(py)$"  PATTERN "jenkins" EXCLUDE)
+INSTALL (FILES     ${CMAKE_CURRENT_SOURCE_DIR}/short_read_connector.sh        DESTINATION bin)
+# INSTALL (FILES     ${CMAKE_CURRENT_SOURCE_DIR}/README.md                      DESTINATION .)
+# INSTALL (FILES     ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE                        DESTINATION .)
 
 # We include the "bin" tag into binary archive file name
 set (CPACK_PACKAGE_FILE_NAME  ${PROJECT_NAME}-${CPACK_PACKAGE_VERSION}-bin-${CMAKE_SYSTEM_NAME})
